components/x-teaser-timeline/src/lib/transform.js

```diff
@@ -28,7 +74,12 @@ const groupItemsByLocalisedDate = (
 	}))
 }
 
-const splitLatestItems = (items, localTodayDate, latestItemsTime) => {
+/**
+ * @param {Array<Item>} items
+ * @param {string} latestItemsTime
+ * @returns {Array<Array<ItemInGroup>,Array<Item>>}
```

Comment:
```suggestion
 * @returns {[ItemInGroup[], Item[]]}
```
Suggestion [non-blocking]: It can be helpful to know that the returned value has a fixed number of items rather than an unbounded list